### ISSUE TITLE
[Bug] False-Positive: Color contrast issue when running on docker

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-service#1219]
Axe core scan engine will produce different scan result when page scanned  in docker container compare to non-container scan. In some cases new color contrast issue is added as false positive. Below example of such occurrence.

Webpages:

https://vi.microsoft.com/en-gb/
https://www.videoindexer.ai/

False positive color contrast issue:

WCAG 1.4.3

Path
`#selectLanguage`

Snippet
`
<select class="form-control" id="selectLanguage" name="selectLanguage" aria-label="Select language">
`

How to fix
`Fix the following: Element has insufficient color contrast of 4.22 (foreground color: #666666, background color: #dddddd, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1`

**Comment by ferBonnin** [comment]
@lamaks created this issue to track the problem reported via email; could you please add more details?

**Comment by ferBonnin** [comment]
@AhmedAbdoOrtiga assigning to you and marking as active since you are investigating this issue

**Comment by ferBonnin** [comment]
Resolving this issue. This has been confirmed to be a real positive that repos in Linux only and not Windows
